src/main/webapp/assets/js/jquery-1.11.1.js

```diff
@@ -0,0 +1,10308 @@
+/*!
```

Comment:
We already have JQuery 3.2.0 in assets/vendors folder. If that newer version of JQuery cannot be used, please move the dependency into assets/vendors folder.